assertk/src/jvmMain/kotlin/assertk/assume.kt

```diff
@@ -0,0 +1,15 @@
+package assertk
+
+/**
+ * Aborts the test instead of failing it, this gives you a way to skip tests based on a dynamic assertion.
+ *
+ * ```
+ * // only run test on windows
+ * assume {
+ *     assertThat(System.getProperty("os.name")).startsWith("Windows")
+ * }
+ * ```
+ */
+fun assume(f: () -> Unit) {
```

Comment:
Should we make this inline like other similar functions to allow suspend functions to be called inside?

Reply:
done